trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -61,21 +63,23 @@ def __defaults__(self):
         self.mid_chord_alignment       = 0.0
         self.thickness_to_chord        = 0.0
         self.blade_solidity            = 0.0
-        self.thrust_angle              = 0.0
-        self.pitch_command             = 0.0 
         self.design_power              = None
         self.VTOL_flag                 = False
         self.design_thrust             = None 
         self.airfoil_geometry          = None
         self.airfoil_polars            = None
         self.airfoil_polar_stations    = None 
         self.radius_distribution       = None
-        self.rotation                  = [1]      # counter-clockwise rotation as viewed from the front of the aircraft 
+        self.rotation                  = 1         # counter-clockwise rotation as viewed from the front of the aircraft
+        self.orientation_euler_angles  = [0.,0.,0.] #s This is X-direction thrust
         self.ducted                    = False         
         self.number_azimuthal_stations = 24
         self.induced_power_factor      = 1.48     # accounts for interference effects
         self.profile_drag_coefficient  = .03     
         self.nonuniform_freestream     = False
+        
+        self.inputs.y_axis_rotation    = 0.
```

Comment:
Yes, if you noticed we removed the vectored thrust network. Instead as an input, rather than an inherent property of the propeller/rotor or of the network, the rotor can rotate (think V-22).

This has an interesting implication, it differentiates rotors (who's default position is vertical) from propellers (who's default is forwards) but allows them to operate in another orientation. This changes how we size, design, and do weight estimation.